scripts/operations/node_management/ncn-image-modification.sh

```diff
@@ -547,6 +578,7 @@ fi
 set_timezone
 if [[ $CSM_RELEASE =~ ^1\.4\.4 ]]; then
   csm_144_patch
+  csm_145_patch
```

Comment:
The regex won't match `CSM_RELEASE=1.4.5` or any prereleases.

This does bring up another question. While we want the contents of the `csm_144_patch` to run for a 1.4.5 release, do we want `csm_145_patch` to run for a 1.4.4 release?

The conditional needs to be broken up to coexist with customers and internal systems that do or don't have access to 1.4.5 yet.

Changing it to this will prevent those using latest release/1.4 docs on a 1.4.4 upgrade/install from pulling in to-new of changes while ensuring both functions run anything 1.4.5 and higher.
```suggestion
if [[ $CSM_RELEASE =~ ^1\.4\.4 ]]; then
  csm_144_patch
elif [[ $CSM_RELEASE =~ ^1\.4\.([5-9]|[1-9]0+|[1-9]{2,}) ]]; then    
    csm_144_patch
    csm_145_patch
```

regex: matches 5-9, every tenth number, and every number that's two digits or more consisting of 1-9. IOW `1.4.5 ≤ 1.4.x < 1.5.0`